src/test/java/HomePageTests.java

```diff
@@ -0,0 +1,20 @@
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+
+public class HomePageTests extends BaseTest {
+
+    @Test (enabled = true, priority = 2, description = "Creating a new playlist")
+    public void newPlaylist() {
+        HomePage homePage = new HomePage(driver);
+        LoginPage loginPage = new LoginPage(driver);
+        loginPage.provideEmail("vlad.kashchuk@testpro.io");
+        loginPage.providePassword("Q4sNKxVN");
+        loginPage.clickSubmit();
+        loginPage.assertSuccessfulLogin();
+        String playlistName = "BrandNew";
+        homePage.createNewPlaylist(playlistName);
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
   
   Please, add Assert

**One test - One Assert**

   
   For example
   
![Clean_Test](https://github.com/vlad199213/A53-QA-Automation-Framework/assets/104761482/2dc82f9e-524d-4d2e-a7e0-808cd3944b92)